### ISSUE TITLE
FIFA22 graphical glitch 

### ISSUE THREAD

**Issue by chico-525** [yuzu-emu/yuzu#7261]
yuzu early access build 2172 vulkan using fidelitity fx
specs: nvidia gt 710 2gddr
          i5-9400f
          8gb ddr4 ram
          256gb ssd
on fifa 22 there is no bug until i clicked play match then I entered and everyting was like from the devs view

![image](https://user-images.githubusercontent.com/84284978/139588890-802d13b6-3c42-4610-9e65-3269c831a3fa.png)
![image](https://user-images.githubusercontent.com/84284978/139588907-6109ddbf-c743-4943-aac0-8b17fd0e91ab.png)
![image](https://user-images.githubusercontent.com/84284978/139588915-8a306e92-76d3-42bc-809f-ad77c6f2875f.png)
![image](https://user-images.githubusercontent.com/84284978/139588926-87a9d8ec-1fa5-4192-b15b-32165b446dee.png)

**Comment by FernandoS27** [comment]
Yeah this requires accelerate DMA for linearToBlockLInear copies.

A temporal workaround: boot the game in GPU Extreme and then change it.

**Comment by Ahmed-E-86** [comment]
> Yeah this requires accelerate DMA for linearToBlockLInear copies.
> 
> A temporal workaround: boot the game in GPU Extreme and then change it.

That fixes the corrupted textures, but there are two more issues:
1- There are no shadows for the players.
2- There is no arrow for the player holding the ball.

**Comment by casasfernando** [comment]
> > Yeah this requires accelerate DMA for linearToBlockLInear copies.
> > A temporal workaround: boot the game in GPU Extreme and then change it.
> 
> @FernandoS27 That fixes the corrupted textures, but there are two more issues: 1- There are no shadows for the players. 2- There is no arrow for the player holding the ball.

Same problems with FIFA 23

**Comment by russdb** [comment]
for what it's worth it happens in ryujinx as well.
